packages/core/streams/openai-stream.ts

```diff
@@ -315,7 +313,11 @@ function createFunctionCallTransformer(
 
       // Stream as normal
       if (!isFunctionStreamingIn) {
-        controller.enqueue(chunk)
+        controller.enqueue(
+          isComplexMode
+            ? textEncoder.encode(getStreamString('text', message))
+            : chunk
```

Comment:
Do we need to encode if we have `experimental_onFunctionCall` but not `experimental_streamData`?

Reply:
Yeah, because all complex mode cares about is which protocol to use. You don't need function handlers to use the protocol.